README.md

```diff
@@ -6,24 +6,25 @@
 
 This project provides a number of libraries to facilitate [ConductR](http://typesafe.com/products/conductr)'s status service and its service lookup service. Note that usage of the libraries in your code is entirely benign when used outside of the context of ConductR i.e. you will find that your applications and services will continue to function normally when used without ConductR. We have also designed the libraries to be a convenience to ConductR's REST and environment variable based APIs, and to have a very low impact on your code.
 
-For the current production release of ConductR (1.0), the libraries use Akka streams/http 1.0 and are structured as follows:
-
-* `"com.typesafe.conductr" %  "conductr-bundle-lib"        % "1.0.2"`
-* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"  % "1.0.2"`
-* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib" % "1.0.2"`
-* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib" % "1.0.2"`
-* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib" % "1.0.2"`
-
-For ConductR 1.1 and above, the libraries use Akka streams/http 2.0 and will be:
-
-* `"com.typesafe.conductr" %  "conductr-bundle-lib"        % "1.2.0"`
-* `"com.typesafe.conductr" %% "java-conductr-bundle-lib"   % "1.2.0"`
-* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"  % "1.2.0"`
-* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "akka24-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play25-conductr-bundle-lib" % "1.2.0"`
+For the current production release of ConductR (1.0), the libraries use Akka streams/http 1.x and are structured as follows:
+
+* `"com.typesafe.conductr" %  "conductr-bundle-lib"         % "1.0.2"`
+* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"   % "1.0.2"`
+* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib"  % "1.0.2"`
+* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib"  % "1.0.2"`
+* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib"  % "1.0.2"`
+
+For ConductR 1.1 and above, the libraries use Akka streams/http 2.x and will be:
+
+* `"com.typesafe.conductr" %  "conductr-bundle-lib"         % "1.2.0"`
+* `"com.typesafe.conductr" %% "java-conductr-bundle-lib"    % "1.2.0"`
+* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"   % "1.2.0"`
+* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib"  % "1.2.0"`
+* `"com.typesafe.conductr" %% "akka24-conductr-bundle-lib"  % "1.2.0"`
+* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib"  % "1.2.0"`
+* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib"  % "1.2.0"`
+* `"com.typesafe.conductr" %% "play25-conductr-bundle-lib"  % "1.2.0"`
+* `"com.typesafe.conductr" %% "lagom10-conductr-bundle-lib" % "1.2.0"`
```

Comment:
It turns out that 1.1 should be fine if we stick to generating service uris instead of acls. @markusjura perhaps we should have an sbt setting for sbt Lagom bundle dictating whether acls or service uris should be generated. Service uris should be the default... Wdyt?

The question becomes whether we can live without acls for the MVP....

@jroper?